plugins/rtmp-services/data/package.json

```diff
@@ -5,7 +5,7 @@
     "files": [
         {
             "name": "services.json",
-            "version": 239
+            "version": 240
```

Comment:
Please also update the version 4 lines above.

Reply:
Updated version